docs/reference/remap.cue

```diff
@@ -360,6 +360,36 @@ remap: {
 			]
 		}
 
+		"Blocks": {
+			href: "blocks"
+
+			description: """
+				TRL supports organizing expressions into blocks using curly braces. Everything
+				inside of a block is evaluated as a single expression. In this example, the value
+				assigned to the variable `$success` is `true` if the value of the `status_code`
+				field is `201`:
+
+				```
+				$success = {
+					$success_codes = [200, 201, 202, 204]
+					$code = .status_code
+					contains($success_codes, $code)
+				}
+				```
+
+				You can also create single-line blocks by separating the expressions with a
+				semicolon (`;`). This block would be the equivalent of the one above:
```

Comment:
This isn't completely accurate.

We use `;` and `\n` as expression terminators, so these are equal:

```rust
$foo = "bar"
.baz = $foo
```

```rust
$foo = "bar"; .baz = $foo
```

However, inside an if-statement, you can put the condition in parentheses, which has the implication that the last expression in those parentheses is resolved to the final condition boolean.

That is to say, this is all valid:

```rust
if false { true } // null
if false || true { true } // true
if ($foo = true; $foo) { true } // true
```